sdk/js/index.ts

```diff
@@ -2,7 +2,6 @@ import {
   Abi,
   Address,
   ByteArray,
-  Log,
```

Comment:
this topics thing is confusing; I'm not the first one to struggle here https://github.com/wagmi-dev/viem/discussions/981 ; I'll have another look

Reply:
Ok so the way I found to make it work was to extend the viem Log type to use args; otherwise you can't use log.args and this is needed to get the stream params (token, streamid etc)

Reply:
Oh neat!